test/spec/browser/request-strategy/JSONP-fallback-per-request.js

```diff
@@ -19,7 +19,8 @@ test('Request strategy uses JSONP fallback on a per requests basis', function(t)
         currentURL.host,
         currentURL.host
       ],
-      timeout: requestTimeout
+      timeout: requestTimeout,
+      protocol: 'http:'
```

Comment:
when calling our mock server, we need to stick with http since it's only running in http. It does not change the test added value